osu.Game.Rulesets.Osu/UI/OsuPlayfield.cs

```diff
@@ -72,6 +73,11 @@ public OsuPlayfield()
 
             AddRangeInternal(poolDictionary.Values);
 
+            // These results share existing judgement pools, so they're only added after the pools are added to the draw hierarchy above.
+            poolDictionary.Add(HitResult.LegacyGreatNoCombo, poolDictionary[HitResult.Great]);
+            poolDictionary.Add(HitResult.LegacyOkNoCombo, poolDictionary[HitResult.Ok]);
+            poolDictionary.Add(HitResult.LegacyMehNoCombo, poolDictionary[HitResult.Meh]);
```

Comment:
Why does this exist given the `DrawableJudgement` change that remaps hit results? If the legacy hit types are to reuse the non-legacy judgement pools, then why bother adding handling to `DrawableJudgement`?